dashboard-app/src/app/pages/hotel-info/views/hotel-info.html

```diff
@@ -150,24 +150,23 @@
                             <label for="checkIn">Check-in</label>
                                 <div class="input-group" moment-picker="hotel.checkIn" format="HH:mm:ss">
                                     <span class="input-group-addon">
-                                        <i class="octicon octicon-clock"></i>
+                                        <i class="fa fa-clock-o" class="fa-lg"></i>
```

Comment:
Don't need to add twice class attribute. It is enough to define in one. eg. class="fa fa-lg fa-clock-o".